lib/logs.coffee

```diff
@@ -40,29 +40,26 @@ deviceModel = require('./models/device')
 # - `line`: when a log line is received.
 # - `error`: when an error happens.
 #
-# @param {String} deviceName - device name
+# @param {String} uuid - device uuid
 # @returns {Promise<EventEmitter>} logs
 #
 # @todo
 # We should consider making this a readable stream.
 #
 # @example
-# resin.logs.subscribe('MyDevice').then (logs) ->
+# resin.logs.subscribe('7cf02a62a3a84440b1bb5579a3d57469148943278630b17e7fc6c4f7b465c9').then (logs) ->
 # 	logs.on 'line', (line) ->
 # 		console.log(line)
 #
 # @example
-# resin.logs.subscribe 'MyDevice', (error, logs) ->
+# resin.logs.subscribe '7cf02a62a3a84440b1bb5579a3d57469148943278630b17e7fc6c4f7b465c9', (error, logs) ->
 # 	throw error if error?
 # 	logs.on 'line', (line) ->
 # 		console.log(line)
 ###
-exports.subscribe = (deviceName, callback) ->
-	Promise.props
-		uuid: deviceModel.get(deviceName).get('uuid')
-		pubNubKeys: configModel.getPubNubKeys()
-	.then (results) ->
-		return logs.subscribe(results.pubNubKeys, results.uuid)
+exports.subscribe = (uuid, callback) ->
+	deviceModel.get(uuid).then(configModel.getPubNubKeys).then (pubNubKeys) ->
```

Comment:
@Page- It is used in both cases to determine the existence of the uuid (since given an invalid uuid to `.get()` throws an error). I could use `.has()` which should express the intention clearer, but that would require me to check the return value of `.has()`, and it's false, throw the error myself. What do you think?